### ISSUE TITLE
capstone version doesnot support MIPS instruction groups.

### ISSUE THREAD

**Issue by zaipingy** [angr/angr#819]
what can i do when i import angr below information show:
```
 angr.analyses.disassembly_utils | Your verison of capstone does not support MIPS instruction groups.
```

i want use angr to analysis a mips firmware. Can anyone help me？ i have update the capstone to 3.0.5.But it doesn't work.

**Comment by zardus** [comment]
To avoid any versioning headaches, can you grab the angr docker container `docker pull angr; docker run -it angr/angr`, and see if this works there? If it does, then check out how that container is built (dockerfile from github.com/angr/angr-dev).

**Comment by ltfish** [comment]
- Not supporting MIPS instruction groups is usually not a problem if you want to analyze MIPS firmware. It is only a problem for packages relying on the `Disassembly` analysis, as it cannot tell whether an instruction is a call/branch or not without the instruction groups support.
- The MIPS instruction groups support only exists in capstone v4. You'll want to manually install [the `next` branch](https://github.com/aquynh/capstone/tree/next) of the capstone engine.

**Comment by zaipingy** [comment]
@zardus Thx! I install the angr by docker and this problem doesn't show again and now i need to learn angr and docker as the same time....

**Comment by zardus** [comment]
Haha, good luck :-)

**Comment by Bennoli13** [comment]
I found this error while trying run angr-mangement in docker shellphish/mechaphish and also error "cannot connect to X server unix:0. Is there any different between docker angr/angr and shellphish/mechaphish that made this angr-management cant run or the problem comes from angr-management files? Here I attached my error. Btw I tried to run angr-management before using virtualenv and it works but I don't know how to solve this error in docker. Thanks
![1](https://user-images.githubusercontent.com/34600849/35546414-1b33e92e-05af-11e8-9cf8-61287604b7a6.PNG)
